atomic_reactor/plugins/pre_resolve_remote_source.py

```diff
@@ -129,6 +132,8 @@ def set_worker_params(self, source_request, remote_source_url, remote_source_con
         override_build_kwarg(self.workflow, 'remote_source_url', remote_source_url)
         override_build_kwarg(self.workflow, 'remote_source_build_args', build_args)
         override_build_kwarg(self.workflow, 'remote_source_configs', remote_source_conf_url)
+        override_build_kwarg(self.workflow, 'remote_source_content_manifest',
```

Comment:
I think this will also require an osbs-client update (add `remote_source_content_manifest` to user_params). It probably doesn't need to be done right away but only when the new plugin is added, is that the plan?

Ideally, `remote_source_content_manifest` should also be added here: [atomic_reactor/schemas/user_params.json](../../tree/master/atomic_reactor/schemas/user_params.json), but since all the other `remote_source_*` params are missing, it would just create an inconsistency.

Reply:
@chmeliik; Exactly; this won't be merged until other work is completed, so the osbs-client work is contingent upon all of that :)

I'll open a separate PR to add the `remote_source_*` params to the schema